packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -71,6 +73,7 @@ export const makeBidSpecShape = (currencyBrand, collateralBrand) => {
   return M.splitRecord(
     { want: collateralAmountShape },
     {
+      exitAfterBuy: true,
```

Comment:
why is `exitAfterBuy: false` invalid?

if it truly is then make it the same in the TS type,
```
  exitAfterBuy?: true,
```

Reply:
I think you're right that the mistake is here. `exitAfterBuy` is a boolean that defaults to `false`.